### PR TITLE
Homepage Added

### PR DESCRIPTION
**Pull request by ashifkhn**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,3 @@
...
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+.App{
+  background-color: black;
+}
```
Comment (vikrantkode) [inline src/App.css:3]: try not using hardcode colors

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import './App.css';
+import { Header } from './components/Header';
+import { HomePage } from './pages/HomePage/HomePage';
+import { QuizPage } from './pages/Quiz/QuizPage';
+import { ResultPage } from './pages/Result/ResultPage';
+
```
Comment (vikrantkode) [inline src/App.js:7]: try importing from one index.js

**Review by vikrantkode — APPROVED** [review]
- In Ui.. try to minimise the width of navbar
- rest is good to go
- Some suggestion given have a look

File: src/App.css (hunk 1 above)
Comment (ashifkhn) [reply src/App.css:3]: I'll change this in the next pr.

File: src/App.js (hunk 2 above)
Comment (ashifkhn) [reply src/App.js:7]: I'll change this in the next pr.

**Review by Pratham82 — APPROVED** [review]
1. Remove commented code.
2. Remove files which are not required like App.test.js.
Apart from that everything looks good, you can merge this.

**Comment by ashifkhn** [comment]
> 1. Remove commented code.
> 2. Remove files which are not required like App.test.js.
>    Apart from that everything looks good, you can merge this.

Will change the suggested things in next pr.

**Pull request merged**